api/proto/node/node.proto

```diff
@@ -33,6 +33,23 @@ service Retrieval {
 	rpc GetBlobHeader(GetBlobHeaderRequest) returns (GetBlobHeaderReply) {}
 	// Retrieve node info metadata
 	rpc NodeInfo(NodeInfoRequest) returns (NodeInfoReply) {}
+
+	////////////////////////////////////////////////////////////////////////////////////////////////
+	// These RPCs are a work in progress and are not yet fully functional. API may not be stable. //
+	////////////////////////////////////////////////////////////////////////////////////////////////
+
+	// GetChunks retrieves all of the chunks from a specified quorum for a blob held by the node.
+	// Eventually this will replace RetrieveChunks.
+	rpc GetChunks(GetChunksRequest) returns (GetChunksReply) {}
+	// GetChunk retrieves a specific chunk for a blob custodied at the Node.
+	rpc GetChunk(common.ChunkKey) returns (common.ChunkData) {}
+	// GetHeader is similar to RetrieveChunks, this just returns the header of the blob.
```

Comment:
You may not want to refer to `RetrieveChunks` in this new set of apis

Reply:
I removed the reference to `RetrieveChunks`, but left the note that it will eventually replace `GetBlobHeader`.